torch/distributed/internal_rpc_utils.py

```diff
@@ -4,30 +4,125 @@
 import copyreg
 import io
 import pickle
+import six
```

Comment:
This adds a dependency on `six` which wasn't present before.

You should use instead https://github.com/pytorch/pytorch/blob/master/torch/_six.py